projects/go-lib/src/lib/components/go-layout/go-layout.component.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../../styles/variables';
+@import '../../../styles/mixins';
```

Comment:
This addition is actually needed, it fixes an issue where the `z-index` on the loader never get's set to the proper value because the mixin for `z-index()` was never imported.